packages/puppeteer-core/src/bidi/Page.ts

```diff
@@ -882,12 +940,40 @@ export class BidiPage extends Page {
     throw new UnsupportedOperation();
   }
 
-  override cookies(): never {
-    throw new UnsupportedOperation();
-  }
-
-  override setCookie(): never {
-    throw new UnsupportedOperation();
+  override async setCookie(
+    ...cookies: Protocol.Network.CookieParam[]
+  ): Promise<void> {
+    for (const cookie of cookies) {
```

Comment:
What is expected behavior in case of one of the cookie failed to be set?

Reply:
I think there is not much we can do except for some pre-validation for common cases (if it is common)